docs: replace <docs-code> block with standard fenced code block for t…

MeAkib opened the pull request:
…ypescript example

Replaced the <docs-code> wrapper with a Markdown fenced code block to improve copy/paste usability, syntax highlighting consistency, and alignment with current documentation formatting standards.

Inspired by #65043

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/contributing-docs/commit-message-guidelines.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.dev application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

alan-agius4 commented on adev/src/content/reference/migrations/signal-inputs.md:
```diff
@@ -49,7 +49,7 @@ export class MyComponent {
 
 @Component({
-template: `Name: {{name() ?? ''}}`
+template: `Name: {{name() ?? ''}}`,
```
Can you also please replace this with backticks? Also this doesn’t seem to be formatted correctly.

MeAkib replied:
Formatting doesn’t work inside a docs-code block — it automatically reverts to the previous state when linting runs.

alan-agius4 commented on adev/src/content/reference/migrations/signal-inputs.md:
````diff
@@ -45,24 +45,24 @@ export class MyComponent {
 
-<docs-code language="angular-ts" highlight="[[4],[7], [10,12]]">
-import {Component, input} from '@angular/core';
+```
````
This change should be reverted.

MeAkib committed:
docs: replace <docs-code> block with standard fenced code block for typescript example

Replaced the <docs-code> wrapper with a Markdown fenced code block to improve
copy/paste usability, syntax highlighting consistency, and alignment with current
documentation formatting standards.

Inspired by angular#65043

alan-agius4 reviewed (approved):
LGTM, thanks for this.

thePunderWoman commented:
This PR was merged into the repository. The changes were merged into the following branches:

- main: caaa5ec8e69e9d3d550985eb71aea213361195cf
- 20.3.x: ba28305b04e27cdc326bdcfdafbbbb76522fea8f
- 21.0.x: c7df828c4842f2bbb4e51bd41c2f7f508f3a399b
